src/v2/Apps/_AuctionPartnerships2/AuctionPartnershipsApp.tsx

```diff
@@ -1,9 +1,678 @@
+                    {
+                      text: "Prospect Indicated Gallery",
+                      value: "Prospect Indicated Gallery",
+                    },
+                    { text: "Art Dealer", value: "Art Dealer" },
+                  ]}
+                  title="Company Type"
+                  onSelect={selected => setFieldValue("Type__c", selected)}
+                  selected={values.Type__c}
+                  error={errors.Type__c}
+                />
+                <Checkbox
+                  onSelect={selected =>
+                    setFieldValue("B2B_TOU_Active_Consent__c", selected)
+                  }
+                  selected={values.B2B_TOU_Active_Consent__c}
+                  error={!!errors.B2B_TOU_Active_Consent__c}
+                >
+                  <Text variant="md" lineHeight={1}>
+                    I agree to Artsy{" "}
+                    <a href="/terms" target="_blank">
+                      Terms of Use
+                    </a>{" "}
+                    and{" "}
+                    <a href="/privacy" target="_blank">
+                      Privacy Policy
+                    </a>
+                    , and to receive emails from Artsy.
+                  </Text>
+                </Checkbox>
+                <Button type="submit" loading={isSubmitting} width="60%">
+                  Apply
+                </Button>
+              </Join>
+            </Form>
+          )}
+        </Formik>
+      </Column>
+    </GridColumns>
+  )
+}
+
+const GlobalAudienceReach: React.FC = () => {
+  const image = resized(
+    "http://files.artsy.net/images/audience_map/temp.jpeg",
+    {
+      width: 910,
+      height: 387,
+    }
+  )
+
+  return (
+    <GridColumns>
+      <Column span={6}>
+        <ResponsiveBox aspectWidth={2208} aspectHeight={938} maxWidth="100%">
+          <Image
+            src={image.src}
+            width="100%"
+            height="100%"
+            srcSet={image.srcSet}
+            lazyLoad
+            alt=""
+          />
+        </ResponsiveBox>
+      </Column>
+      <Column span={6}>
+        <Text textTransform="uppercase" mb={1} variant="sm">
+          Audience
+        </Text>
+        <Text variant="lg" mb={1}>
+          Reach a Global Audience of 24+ Million Collectors and Art Lovers
+        </Text>
+        <Text variant="sm">
+          Since launching in October 2012, Artsy has had over 24 million unique
+          site visitors from 150+ countries across web and mobile, driven by
+          organic search traffic, as well as growing engagement with the world's
+          most comprehensive art resource, and other sources of collector
+          traffic.
+        </Text>
+      </Column>
+    </GridColumns>
+  )
+}
+
+const BiddingTools: React.FC = () => {
+  // const image = resized(
+  //   "http://files.artsy.net/images/auction-devices/temp.png",
+  //   {
+  //     width: 910,
+  //     height: 403,
+  //   }
+  // )
```

Comment:
Do we still need this?

Reply:
yes, waiting for the image from design.

Reply:
I left a placeholder commented out until I get an image without a background back from them.